dvc/repo/live.py

```diff
@@ -17,7 +17,7 @@ def create_summary(out):
 
     html_path = out.path_info.with_suffix(".html")
 
-    out.repo.plots.write_html(html_path, plots, metrics)
+    out.repo.plots.write_html(html_path, plots, metrics, refresh_seconds=5)
```

Comment:
Are you thinking this would be configurable in dvclive?

Reply:
It could be, but I just implemented it as an optional arg because I wasn't sure about what would be a reasonable value to hardcode. If we decide to hardcode it, the introduced change might be simpler but we would still need to be able to deactivate the refresh.

Reply:
Both points make sense. I guess the discussion would be yet again how to expose this DVCLive options to DVC commands without poluting the args.

For point 1, feels like the kind of customization that you would currently apply with `dvc plots modify` or manually editing the `dvc.yaml`.

> revert back to no refresh before exiting

Sounds like a good idea but I'm not sure how we could implement that.

Reply:
Sounds good. Let's either open a separate issue or wait and see if there's a need.